orchestra/communication/views.py

```diff
@@ -35,5 +37,19 @@ def reject_staffing_request_inquiry(request,
                       {})
     if response is None:
         raise Http404
-    return render(request, 'communication/staffing_request_rejected.html',
-                  {})
+
+    next_path = request.GET.get('next')
+    if next_path:
+        return HttpResponseRedirect(next_path)
+    else:
+        return render(request, 'communication/staffing_request_rejected.html',
+                      {})
+
+
+@login_required
+def available_staffing_requests(request):
+    worker = Worker.objects.get(user=request.user)
+    return render(request, 'communication/available_staffing_requests.html',
+                  {
```

Comment:
does this have a link to go back to all available tasks now if you don't get it?